content/swift/concepts/dictionaries/terms/values/values.md

```diff
@@ -0,0 +1,44 @@
+---
+Title: '.values'
+Description: '.value is used to loop through a collection of values within a dictionary.'
+Subjects:
+  - 'Mobile Development'
+  - 'iOS'
+  - 'Computer Science'
+Tags:
+  - 'Dictionary'
+  - 'Properties'
+  - 'Methods'
+CatalogContent:
+  - 'learn-swift'
+  - 'paths/build-ios-apps-with-swiftui'
+---
```

Comment:
```suggestion
---
Title: '.values'
Description: 'Returns a collection containing all the values of the dictionary.'
Subjects:
  - 'Computer Science'
  - 'iOS'
  - 'Mobile Development'
Tags:
  - 'Dictionary'
  - 'Methods'
  - 'Properties'
CatalogContent:
  - 'learn-swift'
  - 'paths/build-ios-apps-with-swiftui'
---
```

Reply:
The description should be short and should start with a verb.
Arranged the subjects and Tags in an alphabetical manner